testcafe/runner-photos.js

```diff
@@ -0,0 +1,30 @@
+const createTestCafe = require("testcafe");
+let testcafe = null;
+
+createTestCafe("localhost", 1337, 1338)
+  .then(tc => {
+    testcafe = tc;
+    const runner = testcafe.createRunner();
+
+    return (
+      runner
+        .src(["testcafe/tests/photos_crud.js"])
+        .browsers(["firefox:headless"])
+        //.browsers(["firefox"])
+
+        .screenshots(
+          "reports/screenshots/",
+          true,
+          "${DATE}_${TIME}/test-${TEST_INDEX}-${FILE_INDEX}.png"
+        )
+        .run(
+          { selectorTimeout: 200000 },
```

Comment:
I don't want to wait 200 seconds to see if a selector has failed. When the test will fail, they will take ages to run... 

60s is already more than enough, no?

Reply:
The default is 100s.
I changed the timeout because I was having issues on .works server which are much slower, which cause my tests to fall because not all pictures on the page were loaded.
I can probably refine and improve this point anyway.